src/lib/comms/sol-http-server-impl-microhttpd.c

```diff
@@ -181,8 +182,31 @@ sol_http_request_get_method(const struct sol_http_request *request)
     return request->method;
 }
 
+static bool
+set_last_modified_header(struct MHD_Response *response, time_t last_modified)
+{
+    struct tm result;
+    char buf[128];
+    size_t r;
+
+    SOL_NULL_CHECK(gmtime_r(&last_modified, &result), false);
+
+    r = strftime(buf, sizeof(buf), "%a, %d %b %Y %H:%M:%S GMT", &result);
```

Comment:
hum... I've missed this in our other bits, but if we `setlocale()` this will be wrong. Do we have a locale-less version of `strftime()` or one we could explicitly specify the locale and then we use "C"? It's not a blocker for this patch, but we should review this in the future.

eventually microhttp provides code to do that.